src/views/activityBar/pipelineView/PipelineDataProvider.ts

```
@@ -119,11 +162,14 @@ export class PipelineDataProvider extends PaginatedDataProvider {
       }
 
       if (!result || 'error' in result) {
+        if (result?.message && result.message.includes('No project')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)